plugin/trino-hive/src/main/java/io/trino/plugin/hive/s3/TrinoS3FileSystem.java

```diff
@@ -414,7 +414,7 @@ public FileStatus getFileStatus(Path path)
 
         return new FileStatus(
                 getObjectSize(path, metadata),
-                S3_DIRECTORY_OBJECT_CONTENT_TYPE.equals(metadata.getContentType()),
+                metadata.getContentType().startsWith(S3_DIRECTORY_OBJECT_CONTENT_TYPE),
```

Comment:
Should this perhaps be case insensitive?
or perhaps parse this using a proper utility class?

```
MediaType.parse(metadata.getContentType()).type().equals(S3_DIRECTORY_OBJECT_CONTENT_TYPE)
```

Reply:
(edited)